### PR TITLE
RMB-829: Suppress "Table audit_... NOT FOUND" in loadDbSchema()

### PR DESCRIPTION
**Pull request by julianladisch**
Also search a table name in the auditingTableName field
and suppress a "Table ... NOT FOUND" warning if found to
be an auditing table.

### CODE REVIEW

File: cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java [hunk 1]
```diff
@@ -180,27 +180,41 @@ private void loadDbSchema(String schemaPath) {
...
+    boolean isAuditTable = false;
+    for (Table table : dbSchema.getTables()) {
+      if (tname.equalsIgnoreCase(table.getTableName())) {
+        dbTable = table;
+        return InitDbTableResult.TABLE_FOUND;
       }
-      if (dbTable == null) {
-        logger.log(Level.SEVERE, "loadDbSchema loadDbSchema(): Table {0} NOT FOUND", tname);
+      if (tname.equalsIgnoreCase(table.getAuditingTableName())) {
+        isAuditTable = true;
```
Comment (hjiebsco) [inline cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java:210]: Would it be simpler to use `return InitDbTableResult.AUDIT_TABLE_FOUND;` here?

**Review by hjiebsco — APPROVED** [review]

File: cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java (hunk 1 above)
Comment (julianladisch) [reply cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java]: Yes, changed.

File: cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java [hunk 2]
```diff
@@ -180,27 +180,36 @@ private void loadDbSchema(String schemaPath) {
...
+  InitDbTableResult initDbTable() {
+    if (jsonField == null) {
+      logger.log(Level.SEVERE, "loadDbSchema(): No primary table name, can not load");
+      return InitDbTableResult.NO_PRIMARY_TABLE_NAME;
+    }
+    // Remove the json blob field name, usually ".jsonb", but in tests also
+    // ".user_data" etc.
+    String tname = this.jsonField.replaceAll("\\.[^.]+$", "");
+    for (Table table : dbSchema.getTables()) {
+      System.out.println(table.getTableName() + " " + table.getMode());
```
Comment (hjiebsco) [inline cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java:199]: Is this intended to keep?

**Review by hjiebsco — APPROVED** [review]

File: cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java (hunk 2 above)
Comment (julianladisch) [reply cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java:199]: No, good catch!
Thanks!

**Pull request merged**